multivariable-calculus/manipulating-derivatives.md

```diff
@@ -1 +1,67 @@
-# Manipulating partial derivatives
+# Manipulating Partial Derivatives
+
+## Example:
+
+We want to compute $\Delta U$ as a function of $T$ and $P$, so we need to determine:
+
+$$
+dU = \left( \frac{\partial U}{\partial T} \right)_P dT + \left( \frac{\partial U}{\partial P} \right)_T dP
+$$
```

Comment:
A few suggestions for all equations:
* Please change the `$$` to `\begin{equation}` and `\end{equation}` so that they get numbered.
* Use `\d{}` for differentials
* It is optional if you'd like to change this since you have already written it correctly, but this book has a shortcut command `\td` for writing thermodynamic derivatives, so you could do like `\td{}{U}{T}{P}` and get the same as what you have for the first temperature derivative.
```suggestion
\begin{equation}
\d{U} = \left( \frac{\partial U}{\partial T} \right)_P \d{T} + \left( \frac{\partial U}{\partial P} \right)_T \d{P}
\end{equation}
```